selfservice/strategy/password/validator_test.go

```diff
@@ -63,7 +63,6 @@ func TestDefaultPasswordValidationStrategy(t *testing.T) {
 			{id: "asdflasdflasdf", pw: "asdflasdflpiuhefnciluaksdzuföfhg", pass: true},
 		} {
 			t.Run(fmt.Sprintf("case=%d", k), func(t *testing.T) {
-				t.Parallel()
```

Comment:
Did you remove this on purpose?

Reply:
Because... `tc` values are overridden by next value. I fixed this to keep `t.Parallel()` at 80b1725